vmware-event-router/internal/processor/aws_event_bridge.go

```diff
@@ -221,23 +223,32 @@ func (awsEventBridge *awsEventBridgeProcessor) createPutEventsInput(baseEvent []
 		cloudEvent := events.NewCloudEvent(event, eventInfo, awsEventBridge.source)
 		jsonBytes, err := json.Marshal(cloudEvent)
 		if err != nil {
-			return eventbridge.PutEventsInput{}, errors.Wrapf(err, "could not marshal cloud event for vSphere event %d from source %s", event.GetEvent().Key, awsEventBridge.source)
+			return []eventbridge.PutEventsInput{}, errors.Wrapf(err, "could not marshal cloud event for vSphere event %d from source %s", event.GetEvent().Key, awsEventBridge.source)
```

Comment:
Would it be ok to return `nil` instead of the slice?

Reply:
Done :+1: